### PR TITLE
not never -> never

### PR DESCRIPTION
**Pull request by unclejustin**
Not never...always? Perhaps not. Probably just means never, or not ever. Never sounds less grrrrr and more heeeey.

## Test plan

n/a

### CODE REVIEW

**Review by eseliger — APPROVED** [review]

File: doc/dev/background-information/web/web_app.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The `tsx` extension makes certain generic syntax impossible and also enables emm
 
 ### `index.*` files
 
-Index files should not never contain declarations on their own.
+Index files should never contain declarations on their own.
```
Comment (efritz) [inline doc/dev/background-information/web/web_app.md:107]:
```
Index files should sometimes contain declarations on their own.
```

This is technically the correct correction :trollpeek:

**Review by efritz — APPROVED** [review]

File: doc/dev/background-information/web/web_app.md (hunk 1 above)
Comment (eseliger) [reply doc/dev/background-information/web/web_app.md:107]: `Index files may be whatever you imagine them to be.`

Comment (efritz) [reply doc/dev/background-information/web/web_app.md:107]: `Index files?`

Comment (unclejustin) [reply doc/dev/background-information/web/web_app.md:107]: A dark horse has appeared!

Comment (eseliger) [reply doc/dev/background-information/web/web_app.md:107]: `*Throws Apple*`

Comment (efritz) [reply doc/dev/background-information/web/web_app.md:107]: image

**Pull request merged**